### PR TITLE
Changes to make sure statsample works with gsl-nmatrix. Will work smoothly with rb-gsl too.

### PR DESCRIPTION
**Pull request by v0dro**
@agarie statsample is now working smoothly with gsl-nmatrix and rb-gsl.

Once that bug in distribution is solved we'll have a statsample that is completely capable of working without GSL :)

### CODE REVIEW

File: statsample.gemspec [hunk 1]
```diff
@@ -85,5 +85,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'minitest', '~> 5.5'
   s.add_development_dependency 'gettext', '~> 3.1'
   s.add_development_dependency 'mocha', '~> 1.1'
-  # s.add_development_dependency 'gsl-nmatrix', '~> 1.17'
+  s.add_development_dependency 'gsl-nmatrix', '~> 1.17'
```
Comment (agarie) [inline statsample.gemspec]: I don't want to add gsl-nmatrix as a dependency without a major release. Can you remove this line before I merge this PR?

**Comment by agarie** [comment]
As I said in that comment, I don't want to introduce a dependency on GSL without doing a major release. I have to fix that issue with Distribution as well, so remove that line from the gemspec before I can merge this pull request, OK?

**Comment by v0dro** [comment]
sure

**Comment by v0dro** [comment]
ok done

**Comment by agarie** [comment]
Thanks!

**Pull request merged**